bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -190,13 +190,18 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
             var content = new Y.eZ.Content(),
                 version = new Y.eZ.Version(),
                 type = event.contentType,
-                mainLanguageCode = type.get('mainLanguageCode'),
                 host = this.get('host'),
                 user = host.get('app').get('user'),
+                activeViewService = host.get('app').get('activeViewService'),
```

Comment:
You can create a variable `app = host.get('app')` and use it here and above for `user`. Small perf optimization.